_drafts/2017-03-20-osd-needs-better-collaboration-tools.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title:  Open source design needs better collaboration tools
+date:   2017-03-09
+categories:
+  - open-source
+  - communities
+  - collaboration
+  - tools
+  - design
+---
+
+Despite the rising awareness and acceptance of UX design, in particular on the web, it has failed to gain much traction in open source software. If for argument's sake, we take UX design to have started in 1995 when [Don Norman started work for Apple as a "user experience architect"](http://blog.invisionapp.com/a-brief-history-of-user-experience/), even though it has a longer history, then the fact that design has failed to make much impact in the open source world for over 20 years suggests that there are structural and systemic barriers that make design and open source development as compatible to each other as oil and water.
```

Comment:
I have such big unnecessary beef with the origin of UX (and specifically that invision post) I have an article written for it that I just haven't published yet. I'll add it to here as well.